src/java.base/share/classes/jdk/internal/access/JavaNioAccess.java

```diff
@@ -67,6 +67,11 @@ public interface JavaNioAccess {
      */
     ByteBuffer newHeapByteBuffer(byte[] hb, int offset, int capacity, MemorySegment segment);
 
+    /**
+     * Used by {@code sun.nio.ch.Util}.
+     */
+    ByteBuffer allocateDirectTemporary(int cap);
```

Comment:
In Util, the phrase used is "temporary buffer and the methods are named like getTemporaryDirectBuffer. The SharedSecrets transposes the words, don't know if that was intended or not.

Reply:
It was intended but I think they should match. I'll change the on in JavaNioAccess.

Reply:
Changed in https://github.com/openjdk/jdk/pull/22339/commits/9d84aa9cfd718265c72e26eff9434f8817d25d80.